.github/workflows/build_pull_request.yml

```diff
@@ -0,0 +1,28 @@
+name: Build Pull Request
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
```

Comment:
We can ignore `edited` here because it will re-run the workflow when updating titles and descriptions too 😅 [Source](https://github.com/nimblehq/survey-compose/pull/26#discussion_r920942396)

Reply:
Updated @ryan-conway